ui-components/src/global/navbar.scss

```diff
@@ -61,7 +61,8 @@ nav.navbar {
 }
 
 .navbar-image-only-container {
-  height: 50px;
+  // 75px - .5rem of padding from the container around it.
+  height: 67px;
```

Comment:
I realize this seems like a magic odd number. I'm up for rounding up to 4.25rem or 75px here if you prefer. 75px for the total height just looked nice, but we can be a bit flexible